ORGANISATIONS.yaml

```
@@ -73,6 +73,18 @@ ifb:
     avatar: "/training-material/shared/images/ifb.png"
     github: false
 
+Australian-BioCommons:
+    name: Australian BioCommons
+    url: https://www.biocommons.org.au/
+    avatar: "/training-material/shared/images/Australian-Biocommons-Favicon-RGB.png"
+    github: false   
```

I noticed https://github.com/AustralianBioCommons/ exists and is the right organisation, since it does y'all could consider changing it like this:

```suggestion
AustralianBioCommons:
    name: Australian BioCommons
    url: https://www.biocommons.org.au/
    avatar: "/training-material/shared/images/Australian-Biocommons-Favicon-RGB.png"
```

and then the github account will be linked to as well